github-management/org-owners-guide.md

```diff
@@ -209,8 +208,8 @@ steps:
   * All external collaborators are removed
   * All webhooks, apps, integrations or services are removed
   * GitHub Pages are disabled
-  * The repo is marked as archived using [GitHub's archive feature]
   * Remove all teams associated with the repo
+  * The repo is marked as archived using [GitHub's archive feature]
```

Comment:
Teams will need to removed before the repo is archived because GitHub won't allow to update teams for an archived repo.